.github/workflows/ruff.yml

```diff
@@ -11,5 +11,5 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - run: pip install --user ruff==0.0.277
```

Comment:
Let's keep this pinned to a given version, otherwise PRs will randomly start failing again!